makefile

```diff
@@ -31,6 +32,10 @@ O_STANC ?= 0
 INC_FIRST ?= -I src -I $(STAN)src -I $(RAPIDJSON)
 USER_HEADER ?= $(dir $<)user_header.hpp
 
+ifeq ($(origin STAN_THREADS), undefined)
```

Comment:
Could this just be `STAN_THREADS ?= TRUE`

Reply:
I would think the same. `STAN_THREADS ?= true` should do it.